odm2api/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
         - geoalchemy-odm2
         - shapely
         - pandas
-        - psycopg2  # [not win]
+        - psycopg2
```

Comment:
@emiliom I updated the recipe by removing the constraint but I did not bump the build number. That won't trigger the upload. I am fixing that now.